src/lime/extension/Main.hx

```diff
@@ -444,8 +452,7 @@ class Main {
 			createTask ("Update", "update"),
 			createTask ("Build", "build", TaskGroup.Build),
 			createTask ("Run", "run"),
-			//createTask ("Test", "test", TaskGroup.Test),
-			createTask ("Test", "test", new TaskGroup ("test", "Test")),
+			createTask ("Test", "test", TaskGroup.Test),
```

Comment:
I think this was a workaround for a VSCode bug from quite a while ago, so it should no longer be needed.